src/multitask_personalization/envs/pybullet/pybullet_env.py

```diff
@@ -344,6 +344,9 @@ def step(
         self, action: PyBulletAction
     ) -> tuple[PyBulletState, float, bool, bool, dict[str, Any]]:
         """Advance the simulator given an action."""
+        # Toggle whether exploration is allowed.
```

Comment:
Could you explain what the purpose of this here? I thought this was in case the user never allows explore, still let the robot explore with a very small epsilon, but since it's flipping and not setting to True, it doesn't seem to be the reason.